frontend/src/app/AppRouter.tsx

```diff
@@ -41,6 +43,10 @@ export const AppRouter = (): JSX.Element => {
   return (
     <WithSuspense>
       <Routes>
+        <Route
+          path={LANDING_ROUTE}
+          element={<PublicElement element={<LandingPage />} />}
+        />
         <Route
           path={ROOT_ROUTE}
           element={<PrivateElement element={<WorkspacePage />} />}
```

Comment:
Bloody github is not showing a correct code 😢 this line should be `HashRouterElement`.